tests/qxmpppresence/tst_qxmpppresence.cpp

```diff
@@ -231,5 +233,25 @@ void tst_QXmppPresence::testPresenceWithMucSupport()
     serializePacket(presence, xml);
 }
 
+void tst_QXmppPresence::testPresenceWithLastUserInteraction()
+{
+    const QByteArray xml(
+        "<presence to=\"coven@chat.shakespeare.lit/thirdwitch\" "
+                  "from=\"hag66@shakespeare.lit/pda\">"
+            "<idle xmlns=\"urn:xmpp:idle:1\" since=\"1969-07-21T02:56:15Z\"/>"
+        "</presence>");
+
+    QXmppPresence presence;
+    parsePacket(presence, xml);
+    QVERIFY(!presence.lastUserInteraction().isNull());
+    QVERIFY(presence.lastUserInteraction().isValid());
+    QCOMPARE(presence.lastUserInteraction(), QDateTime({1969, 7, 21}, {2, 56, 15}, Qt::UTC));
```

Comment:
Forgive my no-being-up-to-date-with-modern-C++, but what compilers support this syntax?

Reply:
Every C++14 compiler:
https://en.cppreference.com/w/cpp/language/list_initialization
https://en.cppreference.com/w/cpp/language/overload_resolution#Implicit_conversion_sequence_in_list-initialization

... but I just noticed that you you're still on C++11: https://github.com/qxmpp-project/qxmpp/blob/master/CMakeLists.txt#L12
I can change it to the old syntax if that's a problem.

Reply:
Yes please, let's use the old syntax. I don't really want to deal with issues from users saying "hey my build broke".